modules/blog/api/fetchBlogPosts.ts

```diff
@@ -0,0 +1,29 @@
+import { BlogPost } from '../types/blogPost';
+
+
+export async function fetchBlogPosts(): Promise<BlogPost[]> {
+  // these are not just mocks anymore, but the actual blog posts that should be shown in production
+  return require('./mocks/blogPosts.json');
```

Comment:
I know you didn't write this, just moved the code, but we should probably update this so our actual blog posts aren't coming from a mocks file 🤔

Reply:
done